### PR TITLE
RHICOMPL-417 - Fix compliance rule expand for Search

### PR DESCRIPTION
**Pull request by dLobatog**
Tests for https://github.com/RedHatInsights/frontend-components/pull/275

### CODE REVIEW

**Comment by bastilian** [comment]
@dLobatog there seems to be a linting error: https://travis-ci.org/RedHatInsights/frontend-components/builds/612289225#L1104

**Comment by dLobatog** [comment]
@bastilian :see_no_evil:  - I just updated the PR to add the missing semicolon

File: packages/inventory-compliance/src/SystemRulesTable.test.js [hunk 1]
```diff
@@ -105,4 +105,19 @@ describe('SystemRulesTable component', () => {
...
+                profileRules={ profileRules }
+                loading={ false }
+                system={ system }
+            />
+        );
+        const instance = wrapper.instance();
+        await instance.setInitialCurrentRows();
+        await instance.updateFilter(false, [ 'high', 'medium' ], []);
+        expect(wrapper.state('currentRows').length / 2).toEqual(1);
+        expect(wrapper.state('currentRows')[1].parent).toEqual(0);
```
Comment (akofink) [inline packages/inventory-compliance/src/SystemRulesTable.test.js]: Sometimes the behavior appears correct for one or two rows, but then another row will be messed up. Maybe we could go ahead and check that this is correct in a loop for `profileRules.length`

Comment (dLobatog) [reply packages/inventory-compliance/src/SystemRulesTable.test.js]: There are only 2 rules in the sample request so I'll have to change that

Comment (akofink) [reply packages/inventory-compliance/src/SystemRulesTable.test.js]: Yeah, I noticed that. :/ You could just set `profileRules = profileRules.concat(profileRules, profileRules, profileRules, ...)`

**Comment by dLobatog** [comment]
I'll make the Fixtures file bigger to reflect reality a bit better, with 20 rules or so. With that, it should be much easier to test for these kind of things.

File: packages/inventory-compliance/src/SystemRulesTable.test.js [hunk 2]
```diff
@@ -89,20 +92,83 @@ describe('SystemRulesTable component', () => {
...
         expect(wrapper.state('currentRows')[0].cells[TITLE_COLUMN].original).
         toEqual('Use direct-lvm with the Device Mapper Storage Driver');
         expect(wrapper.state('currentRows')[2].cells[TITLE_COLUMN].original).
         toEqual('Enable the Docker service');
         await instance.onSort(null, TITLE_COLUMN + 2, SortByDirection.asc);
-        expect(wrapper.state('currentRows').length / 2).toEqual(2);
+        expect(wrapper.state('currentRows').length / 2).toEqual(52);
         expect(wrapper.state('currentRows')[0].cells[TITLE_COLUMN].original).
-        toEqual('Enable the Docker service');
+        toEqual('Add nodev Option to /dev/shm');
```
Comment (dLobatog) [inline packages/inventory-compliance/src/SystemRulesTable.test.js]: After adding more results to the response, the first alphabetical row title has changed

**Pull request merged**